[new-product] Zerto

adrianwells opened the pull request:
Attempt to add Zerto product page

adrianwells committed:
Create zerto.md

Attempt to add Zerto product page

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+releaseDateColumn: true
+eolColumn: End of Support Life (EOSL)
+
+-   releaseCycle: "10.0 Update 2"
```
```suggestion
releases:
-   releaseCycle: "10.0 Update 2"
```

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+    eol: 2024-11-28
+    link: "https://help.zerto.com/bundle/RN.HTML.10.0_U2/page/what_s_new_in_zerto_10_0_update_2.html"
+
+-   releaseCycle: "10.0 Update 1"
```
```suggestion
-   releaseCycle: "10.0_u1"
    releaseLabel "10.0 Update 1"
```

`releaseCycle` is used in API URLs, so it's better to have a `releaseCycle` without spaces and all lowercase. And this notation is also used on https://help.zerto.com/bundle/RN.HTML.10.0_U1/page/what_s_new_in_zerto_10_0_update_1.html, so would be better to have both IMO. Can you update the other releases too ?

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+title: Zerto
+category: app
+tags: zerto
+iconSlug: # N/A currently
```
```suggestion
```

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+iconSlug: # N/A currently
+permalink: /zerto
+releasePolicyLink: https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/product_version_lifecycle_matrix_for_zerto.html
+activeSupportColumn: End of Bug Fixes
```
```suggestion
activeSupportColumn: General Support
```

The column name designate the phase on endoflife, not the date itself.

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+activeSupportColumn: End of Bug Fixes
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of Support Life (EOSL)
```
```suggestion
eolColumn: End of Support Life (EOSL)
```

According to https://dalzssp01.vmware-solutions.cloud.ibm.com/Help/ZSSP/Content/Lifecycle_Matrix/lifecycle_support_summary.htm there is no more updates in this phase. So I think the `eol` column must be the `activeSupport` column (se the End of Support Life column must be removed).

adrianwells replied:
@marcwrobel Thanks for this feedback, however it is not clear yet what should be done here. Is this a recommendation to remove the "eol" values and "eolColumn: End of Support Life (EOSL)"?

marcwrobel replied:
Not removing the `eolColumn`, but moving the `activeSupportColumn` to the `eolColumn`, e.g. moving all `support` value to `eol` and then removing the `activeSupportColumn`.

The reasoning behind is that according to https://dalzssp01.vmware-solutions.cloud.ibm.com/Help/ZSSP/Content/Lifecycle_Matrix/lifecycle_support_summary.htm no fix will ever be release after the End of General Support.

adrianwells replied:
@marcwrobel Thank you for clarifying. I think that can work and have updated accordingly.

marcwrobel replied:
@adrianwells, I did not understood correctly the support policy : there is indeed two phases. Will add the active support column back.

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+    releaseDate: 2019-04-26
+    support: 2020-05-30
+    eol: 2021-05-30
+    link: "https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/product_version_lifecycle_matrix_for_zerto.html"
```
```suggestion
```

The link must be removed if there is not changelog / release note for a release (same applies for version 5.x and 6.x).

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+
+> [Zerto](https://www.zerto.com/zerto-platform/overview/) is a commercial disaster recovery
+> and data protection across on-premises, hybrid, and multi-cloud environments. It provides
+> backup, restore and replication functionality for virtualized or cloud-based workloads.
```
```suggestion
> [Zerto](https://www.zerto.com/zerto-platform/overview/) is a commercial product that provides disaster recovery, ransomware resilience and workload mobility software for virtualized infrastructures and cloud environments.
```

This sentence looks odd: `is a commercial disaster recovery and data protection across on-premises...`. May be this (inspired by https://en.wikipedia.org/wiki/Zerto) ?

marcwrobel commented on products/zerto.md:
```diff
@@ -0,0 +1,106 @@
+
+As each new Zerto version is released, older versions approach their
+End of Bug Fixes and End of Support Life:
+
```
```suggestion
```

This sentence does not add any new information (phases are described below).

marcwrobel reviewed (changes requested):
Thanks for this contribution @adrianwells. Here are my comments and suggestions.

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/zerto.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update zerto.md

Edits based on feedback provided https://github.com/endoflife-date/endoflife.date/pull/4241

adrianwells committed:
Update zerto.md

Correct formatting for "releaseLabel"

adrianwells committed:
Update zerto.md

Updated based on discussion https://github.com/endoflife-date/endoflife.date/pull/4241#discussion_r1436474402

adrianwells reviewed (commented):
@marcwrobel I think this is ready now

marcwrobel committed:
formatting

marcwrobel committed:
drop unnecessary releaseLabel

marcwrobel committed:
add back active support column

marcwrobel committed:
update description

marcwrobel reviewed (approved):
LGTM, merging.

Thanks a lot @adrianwells and sorry for having misunderstood Zerto support policy.
I added the `activeSupportColumn` back and updated the description accordingly.
